pages/index.js

```diff
@@ -222,14 +232,17 @@ export default function Home({ total, today }) {
           />
         )}
 
-        <h2 className="tracking-tight sm:tracking-tight flex sm:flex-row items-center justify-between flex-col">
+        <h2 className="flex flex-col items-center justify-between tracking-tight sm:tracking-tight sm:flex-row">
           <span className="text-4xl font-bold text-secondary-high dark:text-secondary-low">
             LinkFree
           </span>
           <span className="text-2xl dark:text-primary-low text-primary-medium">
             100% Open Source
           </span>
         </h2>
+        <div className="mt-10 md:mt-20">
+          <UserProfile data={data} />
+        </div>
```

Comment:
These margins will be displayed when there is no profile to show, I suggest checking for data first (please rename to `randomProfile`)


```suggestion
        {data && <div className="mt-10 md:mt-20">
          <UserProfile data={data} />
        </div>}
```

Reply:
Done!